Sources/TuistSigning/Certificate/CertificateParser.swift

```diff
@@ -91,3 +91,20 @@ final class CertificateParser: CertificateParsing {
         try System.shared.capture("openssl", "x509", "-inform", "der", "-in", path.pathString, "-noout", "-subject")
     }
 }
+
+private extension String {
+    func sanitizeEncoding() -> String {
+        // we had certificates where encoding in the name was broken - e.g. \\xC3\\xA4 instead of ä
```

Comment:
I'd rephrase it to be more generally written (no-one will know who `we` is in a few months)

Also, are we sure the following regex is bulletproof? Have you tried to look into why we even get the bad encoding in the first place?

Reply:
If we decide to keep this function, I'd add some test case for this.

Reply:
In hat specific case it seems that there is already some encoding mistake in the DB of Apple.
When opening that team in AppStoreConnect the `ä` in the company's name is _broken_. Funnily it looks ok in developer.apple.com and the AppStore.

Fixing that would involve a big financial institution and Apple - so I'm not that optimistic about being able to fix the root cause.

But, yes will add tests